pkg/descheduler/strategies/pod_lifetime_test.go

```diff
@@ -245,12 +253,34 @@ func TestPodLifeTime(t *testing.T) {
 			pods:                    []v1.Pod{*p12, *p13},
 			nodes:                   []*v1.Node{node1},
 			expectedEvictedPodCount: 1,
+			testFunc:                func(podList []v1.Pod) {},
```

Comment:
There's no need for `testFunc` which sets the `DeletionTimestamp` only for pods `p12` and `p13`. Can you create pods `p14` and `p15` instead as copies of `p12` and `p13` and set the `DeletionTimestamp` explicitly?

Reply:
Hi @ingvagabund  thanks for the reivew~~
I think `testFunc` is more easy to extend on the test case. This time we only enhance for DeletionTimestamp, maybe next time we need to do other actions on the Pod. So for me, testFunc is more like a `BuildTestNode` or `BuildTestPod`.

Reply:
@JaneLiuL that's not bad thinking to think ahead, but we don't have any use cases or evidence that we'll need more extensibility at this time. If we do come across that, the test can be refactored then, but for now adding a testFunc is unnecessary

Reply:
@ingvagabund  @damemi  Thanks for your comment~
Then I will try to remove the testFunc, and create the new testPod with deleteTimestamp, is it better?

Reply:
Already remove `testFunc`, may I click `Resolve conversation`?

Reply:
Yeah this is looking a lot cleaner now, thanks!